tests/auth_tests/templates/registration/password_reset_email.html

```diff
@@ -1 +1,2 @@
 {{ protocol }}://{{ domain }}/reset/{{ uid }}/{{ token }}/
+The extra email context can be used here {{ extra_email_context }}
```

Comment:
Please use something like `Extra email context: "{{ extra_email_context }}"` so you can test for the complete string so the test will work as a regression for my correction in views.py.